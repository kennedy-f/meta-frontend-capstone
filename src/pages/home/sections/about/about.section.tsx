import { Container, Grid } from "@mui/material";
import React from "react";

export function AboutSection() {
  return (
    <Container>
      <Container
        maxWidth={"lg"}
        sx={{ color: "#000", height: { md: 400, xs: 800 } }}
      >
        <Grid container>
          <Grid item xs={12} md={6}>
            <article>
              <h3 style={{ fontSize: 40, marginBottom: 0, color: "#000000" }}>
                Little Lemon
              </h3>
              <h4 style={{ fontSize: 20, marginTop: 0 }}> Chicago </h4>
              <p>
                Lorem ipsum dolor sit amet, consectetur adipisicing elit.
                Dolorum esse reiciendis reprehenderit? Dolore dolorem doloribus
                dolorum excepturi itaque maxime modi non nulla possimus, quia
                quisquam saepe sed, veritatis voluptas voluptatum.
              </p>
            </article>
          </Grid>
          <Grid item xs={12} md={6}>
            <div style={{ position: "relative" }}>
              <img
                src={"/imgs/Mario and Adrian b.jpg"}
                alt={"food"}
                height={250}
                width={350}
                style={{
                  position: "absolute",
                  top: 130,
                  right: -15,
                  borderRadius: 20,
                }}
              />
              <img
                src={"/imgs/Mario and Adrian A.jpg"}
                alt={"food"}
                height={250}
                width={350}
                style={{
                  top: 30,
                  position: "absolute",
                  borderRadius: 20,
                  left: 0,
                }}
              />
            </div>
          </Grid>
        </Grid>
      </Container>
    </Container>
  );
}
