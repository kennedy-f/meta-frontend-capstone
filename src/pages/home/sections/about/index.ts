export * from "pages/home/sections/about/about.section";
