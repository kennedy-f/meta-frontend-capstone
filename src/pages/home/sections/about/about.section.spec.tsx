import { render, screen } from "@testing-library/react";
import React from "react";
import { AboutSection } from "./about.section";

describe("about.section", () => {
  it("Should render", () => {
    const { container } = render(<AboutSection />);
    expect(container).toMatchSnapshot();
  });

  it("Should render texts", () => {
    render(<AboutSection />);
    expect(screen.getByText(/Little Lemon/)).toBeInTheDocument();
    expect(screen.getByText(/Chicago/)).toBeInTheDocument();
    expect(
      screen.getByText(
        /Lorem ipsum dolor sit amet, consectetur adipisicing elit./
      )
    ).toBeInTheDocument();
  });
});
