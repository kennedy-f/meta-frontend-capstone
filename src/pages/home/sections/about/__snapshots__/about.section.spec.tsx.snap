// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about.section Should render 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-ot339a-MuiContainer-root"
    >
      <div
        class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6 css-1fyyp8j-MuiGrid-root"
        >
          <article>
            <h3
              style="font-size: 40px; margin-bottom: 0px; color: rgb(0, 0, 0);"
            >
              Little Lemon
            </h3>
            <h4
              style="font-size: 20px; margin-top: 0px;"
            >
               Chicago 
            </h4>
            <p>
              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dolorum esse reiciendis reprehenderit? Dolore dolorem doloribus dolorum excepturi itaque maxime modi non nulla possimus, quia quisquam saepe sed, veritatis voluptas voluptatum.
            </p>
          </article>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6 css-1fyyp8j-MuiGrid-root"
        >
          <div
            style="position: relative;"
          >
            <img
              alt="food"
              height="250"
              src="/imgs/Mario and Adrian b.jpg"
              style="position: absolute; top: 130px; right: -15px; border-radius: 20px;"
              width="350"
            />
            <img
              alt="food"
              height="250"
              src="/imgs/Mario and Adrian A.jpg"
              style="top: 30px; position: absolute; border-radius: 20px; left: 0px;"
              width="350"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
